Makefile

```diff
@@ -31,3 +32,16 @@ format-check:
 	npx prettier $(JS_FILES) $(CSS_FILES) --check
 	npx eslint $(JS_FILES)
 	@echo "format-check passed"
+
+deploy: freeze
+	git branch -D gh-pages
+	git branch -D $(TEMP_DEPLOY_BRANCH)
+	git checkout -b $(TEMP_DEPLOY_BRANCH)
+	git add -f build
+	git commit -am "Deploy on gh-pages"
+	git subtree split --prefix build -b gh-pages
+	# git push --force "https://${GH_TOKEN}@${GH_REF}.git" $(TEMP_DEPLOY_BRANCH):gh-pages
```

Comment:
remove this line if it is not needed?

Reply:
I was actually in a doubt since the remote might not always be called `origin`. Like right now it pushes the build folder to gh-pages on origin remote. But that might not be the case always

Reply:
It's fine, you can keep this.